Makefile

```diff
@@ -1,10 +1,9 @@
 venv:
-	@echo "making a venv in ~/venv/whirlwind"
-	mkdir -p ~/venv
-	virtualenv -p python ~/venv/whirlwind
+	@echo "making a venv in ./.venv"
+	python3 -m venv .venv
```

Comment:
This is more modern?  `venv` is part of standard library since Python 3.3.

Reply:
Yes, and it is the recommended method since v3.5 (2015). Although it is a subset of virtualenv (thus limitations), 99.9% we just use it to create venvs, plus it also creates the directory structure.
Ref: https://docs.python.org/3/library/venv.html
Changed in version 3.5: The use of venv is now recommended for creating virtual environments.